app/src/main/java/io/gnosis/safe/ui/safe/settings/safe/SafeSettingsViewModel.kt

```diff
@@ -32,7 +32,11 @@ class SafeSettingsViewModel @Inject constructor(
             runCatching {
                 safe?.let {
                     safeRepository.removeSafe(safe)
-                    safeRepository.clearActiveSafe()
+                    val safes = safeRepository.getSafes()
+                    if (safes.isEmpty())
+                        safeRepository.clearActiveSafe()
+                    else
+                        safeRepository.setActiveSafe(safes.first())
```

Comment:
Curly braces for `if/else` which are not conditional assignments! Without, is only ok for assignments:  https://kotlinlang.org/docs/reference/coding-conventions.html#using-conditional-statements

Reply:
I am failing to see why? Kotlin style guide says nothing about it. Is it something you want to have? And if you want to have curly brances it would be consistent to use them always.

Reply:
https://kotlinlang.org/docs/reference/coding-conventions.html#formatting-control-flow-statements That is the right section. I am not sure if we are following the kotlin official code convention. For flow control, even if it's one statement, curly braces apply, if it's a conditional assignment you can lose them. 

This is not something I personally want, I just thought we were using this code conventions. This is not something that would block a PR for me.

Reply:
they talk about `multiline' condition statement. We can agree to always use curly braces except for simple conditional expressions. Would it be ok?

Reply:
You can see in the examples, that single lines have them. But anyway, I just wrote to you, I don't care either way :) for me the only thing blocking this PR was already addressed. The nested scrolling and adding additional assertions to the tests for the safe ordering. That's it, merge away if you like :) Let's not over discuss stuff.

Reply:
no! let's continue our stylistic battle! tabs or spaces?! just kidding...